airflow/providers/neo4j/hooks/neo4j_hook.py

```diff
@@ -0,0 +1,138 @@
+# -*- coding: utf-8 -*-
```

Comment:
@potiuk I can't find our latest "rules" for new naming. Should this be ` airflow/providers/neo4j/hooks/neo4j.py`

Reply:
I think it's the way it should be. Other option I see is `airflow/providers/neo4j/hooks.py` but this can be confusing.

Reply:
```
from airflow.providers.neo4j.hooks.neo4j import Neo4jHook
```

or how about this?

```
from airflow.providers.neo4j.hook import Neo4jHook
```

(`airflow/providers/neo4j/hook.py`) If it's unlikely we'll ever have more than single hook?

Reply:
I am ok with `airflow/providers/neo4j/hook.py`, @potiuk?